tests/lib/rules/v-for-delimiter-style.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * @fileoverview enforce `v-for` directive's delimiter style
+ * @author Flo Edelmann
+ * @copyright 2020 Flo Edelmann. All rights reserved.
+ * See LICENSE file in root directory for full license.
+ */
+'use strict'
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+
+const RuleTester = require('eslint').RuleTester
+const rule = require('../../../lib/rules/v-for-delimiter-style')
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+const tester = new RuleTester({
+  parser: require.resolve('vue-eslint-parser'),
+  parserOptions: { ecmaVersion: 2015 }
+})
+
+tester.run('v-for-delimiter-style', rule, {
+  valid: [
+    {
+      filename: 'test.vue',
+      code: ''
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div v-for="x in xs"></div></template>'
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div v-for="x    in xs"></div></template>'
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div v-for="x in    xs"></div></template>'
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div v-for="x    in    xs"></div></template>'
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div v-for="x in xs"></div></template>',
+      options: ['in']
+    },
+    {
+      filename: 'test.vue',
+      code: '<template><div v-for="x of xs"></div></template>',
+      options: ['of']
+    }
+  ],
+  invalid: [
+    {
+      filename: 'test.vue',
+      code: '<template><div v-for="x of xs"></div></template>',
+      output: '<template><div v-for="x in xs"></div></template>',
+      errors: ["Expected 'in' instead of 'of' in 'v-for'."]
```

Comment:
Could you use an object expression to check the location of the error?

https://eslint.org/docs/developer-guide/nodejs-api#ruletester

```suggestion
      errors: [{
        message: "Expected 'in' instead of 'of' in 'v-for'.",
        column: ?,
      }]
```